.env.example

```diff
@@ -1,2 +1,3 @@
-REACT_APP_AAD_CLIENT_ID=276fd164-c30c-485f-bd76-a8f3b5bcac74
+REACT_APP_AAD_CLIENT_ID=5dd59e6d-1cb1-4ec7-8e26-79a6dd9859b4
+REACT_APP_AAD_TENANT_ID=common
```

Comment:
[See the docs](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/initialization.md#optional-configure-authority) for the auth library we are using. 

This should be set to `common`, which it is by default, if the app registration allows logging in from multi-tenants